packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -99,6 +112,9 @@ export default class TooltipAnchor extends React.Component<Props, State> {
     }
 
     componentWillUnmount() {
+        this._unsubscribeFromTracker && this._unsubscribeFromTracker();
```

Comment:
🤷‍♂️ I'd prefer, makes the intent a little bit clearer:

```
if (this._unsubscribeFromTracker) {
    this._unsubscribeFromTracker();
}
```

Reply:
Yes, however, the old flow complained about that which means capturing the variable locally first and then doing this (not a big deal).. I'll see if it complains now we've updated it and edit accordingly